Build fails without pulseaudio

I am trying to build with --no-default-features as I don't use pulseaudio. However the build fails because unresolved import of libpulse_binding.

Please find below the output on the error:

error[E0432]: unresolved import `libpulse_binding`
 --> src/main.rs:3:5
  |
3 | use libpulse_binding as pulse;
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^ no `libpulse_binding` external crate

warning: unused import: `lazy_static::lazy_static`
  --> src/blocks/sound.rs:30:5
   |
30 | use lazy_static::lazy_static;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: `#[warn(unused_imports)]` on by default

error: aborting due to previous error; 1 warning emitted

--- a/src/blocks/sound.rs
+++ b/src/blocks/sound.rs
@@ -11,6 +11,7 @@ use {
     crate::pulse::proplist::{properties, Proplist},
     crate::pulse::volume::{ChannelVolumes, VOLUME_MAX, VOLUME_NORM},
     crossbeam_channel::unbounded,
+    lazy_static::lazy_static,
     std::cell::RefCell,
     std::cmp::min,
     std::collections::HashMap,
@@ -27,7 +28,6 @@ use std::thread;
 use std::time::{Duration, Instant};
 
 use crossbeam_channel::Sender;
-use lazy_static::lazy_static;
 use serde_derive::Deserialize;
 use uuid::Uuid;
 

--- a/src/main.rs
+++ b/src/main.rs
@@ -1,5 +1,6 @@
 #[macro_use]
 extern crate serde_json;
+#[cfg(feature = "pulseaudio")]
 use libpulse_binding as pulse;
 
 #[macro_use]

